src/Controller/EnMarche/NeedSyncUserInterface.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace AppBundle\Controller\EnMarche;
+
```

Comment:
maybe a little comment for explain for what this empty interface has been created